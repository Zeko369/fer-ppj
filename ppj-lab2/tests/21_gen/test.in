IDN 1 m
OP_PRIDRUZI 1 =
BROJ 1 1214
IDN 2 m
OP_PRIDRUZI 2 =
IDN 2 m
OP_MINUS 2 -
L_ZAGRADA 2 (
BROJ 2 822
D_ZAGRADA 2 )
IDN 3 q
OP_PRIDRUZI 3 =
IDN 3 q
OP_PUTA 3 *
L_ZAGRADA 3 (
IDN 3 q
D_ZAGRADA 3 )
IDN 4 n
OP_PRIDRUZI 4 =
BROJ 4 20
OP_PUTA 4 *
L_ZAGRADA 4 (
IDN 4 m
D_ZAGRADA 4 )
