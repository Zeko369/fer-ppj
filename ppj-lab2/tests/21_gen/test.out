<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 m
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 1214
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 m
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        IDN 2 m
       <T_lista>
        $
      <E_lista>
       OP_MINUS 2 -
       <E>
        <T>
         <P>
          L_ZAGRADA 2 (
          <E>
           <T>
            <P>
             BROJ 2 822
            <T_lista>
             $
           <E_lista>
            $
          D_ZAGRADA 2 )
         <T_lista>
          $
        <E_lista>
         $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 q
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         IDN 3 q
        <T_lista>
         OP_PUTA 3 *
         <T>
          <P>
           L_ZAGRADA 3 (
           <E>
            <T>
             <P>
              IDN 3 q
             <T_lista>
              $
            <E_lista>
             $
           D_ZAGRADA 3 )
          <T_lista>
           $
       <E_lista>
        $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 n
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          BROJ 4 20
         <T_lista>
          OP_PUTA 4 *
          <T>
           <P>
            L_ZAGRADA 4 (
            <E>
             <T>
              <P>
               IDN 4 m
              <T_lista>
               $
             <E_lista>
              $
            D_ZAGRADA 4 )
           <T_lista>
            $
        <E_lista>
         $
     <lista_naredbi>
      $
